composer.json

```diff
@@ -34,22 +33,24 @@
         "colinmollenhour/cache-backend-file": "1.4",
         "colinmollenhour/cache-backend-redis": "1.10.2",
         "colinmollenhour/credis": "1.6",
-        "colinmollenhour/php-redis-session-abstract": "~1.2.2",
-        "composer/composer": "1.4.1",
-        "magento/composer": "~1.2.0",
+        "colinmollenhour/php-redis-session-abstract": "~1.3.8",
+        "composer/composer": "~1.6.0",
+        "friendsofphp/php-cs-fixer": "~2.10.0",
```

Comment:
Why is it moved to `require` from `require-dev`?

Reply:
No reason. Moved back to `require-dev`.